.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "javascript.suggestionActions.enabled": false
```

Comment:
This fix a VSCode issue that shows a warn when a js file has a module.export

See [https://stackoverflow.com/questions/49582984/how-do-i-disable-js-file-is-a-commonjs-module-it-may-be-converted-to-an-es6](https://stackoverflow.com/questions/49582984/how-do-i-disable-js-file-is-a-commonjs-module-it-may-be-converted-to-an-es6)